business-registry-dashboard/app/locales/en-CA.ts

```diff
@@ -482,6 +494,11 @@ export default {
     }
   },
   modal: {
+    manageBusiness: {
+      success: {
+        title: 'Business Added. You can now manage {identifier}.'
```

Comment:
Weird mixed case. I know part of it is the title but then the rest isn't?

Reply:
True. The reason why I have it as is because on success, we're showing a toast. The toast only will have a title (no description), so I squeezed them both here. Let me know if you want me to rename this Sev.

Reply:
Is there a section with a single string for toasts (as this is a modal section)?

What do we do for other toasts?

If there's no better guidance, just leave it like this I guess.

Reply:
(This can be resolved unless you want to chat about it further.)

Reply:
You make a great point Sev. So from what I saw, they're kind of scattered. There are for example `toast.errorResendingAffInvite` and there are `form.manageBusiness.toast.emailSent`. What I did here for now, I changed the `title` to `toast`, that should be good if you're good with that.

Reply:
I'm good with that. Too bad about the scattered existing code.